File: 0191-number-of-1-bits/0191-number-of-1-bits.c
int hammingWeight(int n) 
{
    if(n == 1){ return 1;}
    else 
    {
        if(n%2 == 1)
        {
            return 1 + hammingWeight(n/2);
        }
        else
        {
            return hammingWeight(n/2);
        }
    }
}

File: 0191-number-of-1-bits/0191-number-of-1-bits.py
class Solution(object):
    def hammingWeight(self, n):
        """
        :type n: int
        :rtype: int
        """
        if(n == 0):
            return 0
        elif(n == 1):
            return 1
        elif(n%2):
            return self.hammingWeight(n//2) + 1
        else:
            return self.hammingWeight(n//2)